tests/test_quantum_info_clifford.py

```diff
@@ -17,11 +17,12 @@
 
 
 def construct_clifford_backend(backend):
-    if isinstance(backend, TensorflowBackend):
+    if backend.__class__.__name__ in ["TensorflowBackend", "CuQuantumBackend"]:
```

Comment:
```suggestion
    if isinstance(backend, (TensorflowBackend, CuQuantumBackend)):
```

`isinstance` works even for multiple options (of course they are in OR)

Reply:
I specifically avoided `isinstance` so not to have to import `CuQuantumBackend` from `qibojit` on a `qibo` test. Also because github is not installing `cuquantum` to test, so it would fail anyways.

Reply:
I acknowledge the problem, though I still believe that is not the optimal solution in general. But it could be for this PR.

I already wrote this somewhere else, but it would be better that tests would be shipped together with the code. So, we should face `cuquantum` issues only on the `qibojit` side.
(but we will improve this, starting from `qibo-core`, I just need some time to get back to it)
In `qibojit`, we could gate tests with marks, and possibly mock if it is too entangled.

But all of this is overly complex for this PR, so maybe let's just keep your original solution.